src/Apps/Artist/Routes/AuctionResults/Components/TableSidebar.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react"
+
+import { Flex, Row, Separator } from "@artsy/palette"
+import { AuctionFilters } from "./AuctionFilters"
+
+export const TableSidebar = () => {
+  return (
```

Comment:
E.g., if a user had to jump in here and pluck something from the context, they just add a line break. Avoiding implicit returns saves a lot of ceremony at scale.